### ISSUE TITLE
lazy loading router store does not include params

### ISSUE THREAD

**Issue by FillScheers** [ngrx/platform#1814]
## Minimal reproduction of the bug/regression with instructions:

when lazy loading a module with routing and router parameters.

the router store won't display the parameters because of angulars routing separation

## Expected behavior:

router params should be included

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):

angular 7.2.14
@ngrx/router-store 7.4.0

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[* ] No

**Comment by jtcrowson** [comment]
@FillScheers can you provide reproduction steps?

**Comment by spookgit** [comment]
We thought we had the same problem, with our route params being lost in the lazy loaded routing.

Turns out that our issue was actually with the router serializer that are in the ngrx Router docs, where each child route would overwrite the previous child route and so we'd end up with an empty params object (the last one).

We found a good solution in [this article](https://medium.com/simars/ngrx-router-store-reduce-select-route-params-6baff607dd9), where the serializer merges the params from each child route, rather than overwriting them.

Is this the same problem that you are experiencing?
